### DISCUSSION TITLE
Quantifying Magnitude of Survival Difference

### DISCUSSION THREAD

**Discussion by j-fife** [CamDavidsonPilon/lifelines#1280] [Q&A]
I was wondering if there was a more sound way to measure the magnitude of the difference in survival among two groups. The logrank test is a great way to ensure that two populations are affected at different rates, but is there a way in lifelines to get the difference? Something like an odds ratio or another magnitude measure? 

Thanks

**Comment by CamDavidsonPilon** [comment]
Hi @j-fife, I would recommend the RMST - restricted-mean-survival-time. It's in lifelines under `lifeline.utils.restricted_mean_survival_time`
